natgeo/mortar
# [SYTO-652] Create Video Playlist Interactive Example Page

https://jira.natgeo.com/browse/SYTO-652

#### How to test
1. ``gulp serve``
2. Open http://localhost:3000/components/videoplaylist/index.html
3. Play around with the options(another json endpoint is /mockdata/videoplaylist/moredata.json)
4. Check the changes in the result section.
5. You can also access by the VideoPlaylist configurator link on the components page.
